src/components/talk/talkText.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useRecoilState } from 'recoil';
+import { talkState } from '../../state/state';
+import { TalkBoxInfo } from '../../interfaces/interface';
+const Text = styled.div<TalkBoxInfo>`
+  position: relative;
+  display: inline-block;
+  max-width: 70%;
+  padding: 10px;
+
+  margin-top: 7px;
+  font-size: 15px;
+  border-radius: 13px;
+  background-color: yellow;
+  display: flex;
+  background-color: ${({ IsUser, nowIsUser }) =>
+    IsUser === nowIsUser ? 'yellow' : 'grey'};
+  margin-left: ${({ IsUser, nowIsUser }) =>
+    IsUser === nowIsUser ? 'auto' : '0'};
```

Comment:
```suggestion
  ${({ IsUser, nowIsUser }) => 
    IsUser === nowIsUser ?
    css `
      background-color: 'yellow';
      margin-left: 'auto';
    ` : 
    css `
      background-color: 'grey';
      margin-left: '0';
    `
    };
```

취향차이일 수도 있지만 저는 변수가 두 가지 속성 이상에 조건부로 작용하게 되면 `styled-components`의 `css`를 사용해서 이렇게 묶어주는게 조건에 따라 정의할 수 있어서 깔끔해보이더라구요~!